backend/config/config.go

```diff
@@ -438,3 +438,20 @@ func migrateUserLanguage(appconf *AppConfig) {
 		delete(frontconf, "userLanguage")
 	}
 }
+
+// migrateActiveTokens removes tokens from AccountsConfig.
+func migrateActiveTokens(accountsConf *AccountsConfig) {
```

Comment:
looks like you are not calling this function anywhere yet.

Reply:
oh....yes